bridges/snowbridge/pallets/system/src/lib.rs

```diff
@@ -211,6 +214,12 @@ pub mod pallet {
 		PricingParametersChanged {
 			params: PricingParametersOf<T>,
 		},
+		/// Register token
+		RegisterToken {
+			asset_id: Location,
```

Comment:
https://github.com/Snowfork/polkadot-sdk/pull/128/commits/3621de4f40d4c69d24cfbc98b89757ed343846ec

Reply:
Hmmm so does that mean if xcm is upgraded to version 5, we need to still send v4 Locations because otherwise the asset ID changes? Or what is the impact of a xcm version upgrade?

Reply:
Yeah, the prerequisite of the upgrade is to keep compatible across versions so there should be some convert logic from V4 to V5 and vice-versa, with `VersionedLocation` we're immune to these changes.